pyzebra/comm_export.py

```diff
@@ -0,0 +1,75 @@
+import numpy as np
+
+
+def correction(value, lorentz=True, zebra_mode="--", ang1=0, ang2=0):
+    if lorentz is False:
+        return value
+    else:
+        if zebra_mode == "bi":
+            corr_value = np.abs(value * np.sin(ang1))
+            return corr_value
+        elif zebra_mode == "nb":
+            corr_value = np.abs(value * np.sin(ang1) * np.cos(ang2))
+            return corr_value
+
+
+def export_comm(data, path, lorentz=False):
+    """exports data in the *.comm format
+    :param lorentz: perform Lorentz correction
+    :param path: path to file + name
+    :arg data - data to export, is dict after peak fitting
+
+    """
+
+    align = ">"
+    if data["meta"]["indices"] == "hkl":
+        extension = ".comm"
+        padding = [6, 4, 10, 8]

```

Comment:
These paddings look artificial to me. Do you think it was just tabs between numbers, and the different padding sizes are because of different lengths of those written numbers?

Reply:
I'm sorry, I think that the answer I send from email got stuck somewhere. The files look very organized, so I think its really read line by line in this and first 6 chars correspond to number and so on. Oksana would like to keep it this way.